packages/app-mobile/components/NoteEditor/ImageEditor/promptRestoreAutosave.ts

```diff
@@ -0,0 +1,37 @@
+import { _ } from '@joplin/lib/locale';
+import shim from '@joplin/lib/shim';
+import { Alert } from 'react-native';
+import { clearAutosave, getAutosaveFilepath, readAutosave } from './autosave';
+
+export type RestoreAutosaveCallback = (data: string)=> void;
+
+const promptRestoreAutosave = async (onRestoreAutosave: RestoreAutosaveCallback) => {
+	const autosavePath = getAutosaveFilepath();
+
+	if (await shim.fsDriver().exists(autosavePath)) {
+		const title: string|null = null;
+		const message = _(
+			'An autosaved drawing was found. It can either be discarded or attached to the current note.'
```

Comment:
Maybe just "Joplin has recovered an unsaved drawing"? Then a "Restore it" / "Discard" button below. But I wonder if this message is a bit confusing because what is being restored? What if there are multiple drawings in the note and we overwrite a version of one drawing that the user wanted to keep?

Reply:
The "restore from autosave" functionality should never overwrite an existing drawing. It should always create a copy of the autosaved drawing and attach that to the end of the note (e.g. using `this.attachFile`).

Perhaps "An autosaved drawing was found. Attach a copy of it to the note?".